docroot/sites/all/modules/dev/wsuser/wsuser.module

```diff
@@ -824,20 +824,18 @@ function wsuser_user_delete($account) {
 function wsuser_user_login(&$edit, &$account) {
   $countries = _locationservices_supported_countries();
   $browser_cc = user_location_get_browser_country();
-  $browser_countryname = @$countries[user_location_get_browser_country()];
+  $browser_countryname = @$countries[$browser_cc];
```

Comment:
Would be better with isset($countries[$browser_cc]) ? $countries[$browser_cc] : NULL;